lib/sisimai/bite/email/ezweb.rb

```diff
@@ -0,0 +1,260 @@
+module Sisimai::Bite::Email
+  module EZweb
+    # Sisimai::Bite::Email::EZweb parses a bounce email which created by au EZweb.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/EZweb.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :'from'       => %r/[<]?(?>postmaster[@]ezweb[.]ne[.]jp)[>]?/i,
+        :'subject'    => %r/\AMail System Error - Returned Mail\z/,
+        :'received'   => %r/\Afrom[ ](?:.+[.])?ezweb[.]ne[.]jp[ ]/,
+        :'message-id' => %r/[@].+[.]ezweb[.]ne[.]jp[>]\z/,
+      }
+      Re1 = {
+        :begin  => %r{\A(?:
+             The[ ]user[(]s[)][ ]
+            |Your[ ]message[ ]
+            |Each[ ]of[ ]the[ ]following
+            |[<][^ ]+[@][^ ]+[>]\z
+            )
+        }x,
+        :rfc822   => %r#\A(?:[-]{50}|Content-Type:[ ]*message/rfc822)#,
+        :boundary => %r/\A__SISIMAI_PSEUDO_BOUNDARY__\z/,
+        :endof    => %r/\A__END_OF_EMAIL_MESSAGE__\z/,
+      }
+      ReFailure = {
+        # notaccept: [ %r/The following recipients did not receive this message:/ ],
+        mailboxfull: [
+          %r/The user[(]s[)] account is temporarily over quota/,
+        ],
+        suspend: [
+          # http://www.naruhodo-au.kddi.com/qa3429203.html
+          # The recipient may be unpaid user...?
+          %r/The user[(]s[)] account is disabled[.]/,
+          %r/The user[(]s[)] account is temporarily limited[.]/,
+        ],
+        expired: [
+          # Your message was not delivered within 0 days and 1 hours.
+          # Remote host is not responding.
+          %r/Your message was not delivered within /,
+        ],
+        onhold: [
+          %r/Each of the following recipients was rejected by a remote mail server/,
+        ],
+      }
+      Indicators = Sisimai::Bite::Email.INDICATORS
+
+      def description; return 'au EZweb: http://www.au.kddi.com/mobile/'; end
+      def smtpagent;   return Sisimai::Bite.smtpagent(self); end
+      def headerlist;  return ['X-SPASIGN']; end
+      def pattern;     return Re0; end
+
+      # Parse bounce messages from au EZweb
+      # @param         [Hash] mhead       Message headers of a bounce email
+      # @options mhead [String] from      From header
+      # @options mhead [String] date      Date header
+      # @options mhead [String] subject   Subject header
```

Comment:
__[rubocop]__
* Style/Semicolon: Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)
* Layout/TrailingWhitespace: Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

<!-- Comment by [SideCI](https://sideci.com) -->